package/CHANGELOG

```diff
@@ -76,10 +76,11 @@ Fixes
   * Added missing selection module to leaflet.py (Issue #2612)
 
 Enhancements
+  * Added elements attribute to PDBParser (Issue #2553)
```

Comment:
You should also add 2647 here.

Reply:
Sure!
@IAlibay, I git pulled on my local repo and I can't import the mda universe due a module error. It has issues with importing the tqdm package (it was one of the newly merged changes I believe). Can you help me in figuring out what am I doing wrong? Thanks and sorry for the trouble